src/migration/MigrationExecutor.ts

```diff
@@ -41,6 +52,7 @@ export class MigrationExecutor {
 
         const options = <SqlServerConnectionOptions|PostgresConnectionOptions>this.connection.driver.options;
         this.migrationsTableName = connection.options.migrationsTableName || "migrations";
+        this.migrationsForce = !!connection.options.migrationsForce;
```

Comment:
Connection options should have boolean value, so no `!!` necesarry. Even if someone passed other type it should be parsed as soon as possible to boolean type.

Reply:
All of the `BaseConnectionOptions` are optional (they can also be `undefined`). Removing the `!!` results in a compile error and I'd also have to set the private `migrationsForce` variable to `boolean | undefined`.